cocos/audio/android/AudioDecoder.cpp

```diff
@@ -392,7 +392,6 @@ bool AudioDecoder::decodeToPcm()
     SLMetadataInfo *keyInfo, *value;
     for (i = 0; i < itemCount; i++)
     {
-        keyInfo = nullptr;
```

Comment:
I think we should initialize it when define it before `for`, and `keySize`, `valueSize` can do the same way. 

And i found `value` is not used.